### PR TITLE
docs(v3): add minimum SDK versions for Android

### PR DESCRIPTION
**Pull request by jspizziri**
Created in response to https://github.com/doublesymmetry/react-native-track-player/pull/1667

### CODE REVIEW

File: docs/docs/v3-migration.md [hunk 1]
```diff
@@ -34,3 +34,13 @@ await TrackPlayer.updateOptions({
 -  TrackPlayer.destroy();
 -  TrackPlayer.stop();
 ```
+
+You also meed to have a minimum comile & target SDK Version for Android of 31:
```
Comment (mpivchev) [inline docs/docs/v3-migration.md:38]:
```suggestion
You also meed to have a minimum comile & target SDK of 31 (Android 12)
```

File: docs/versioned_docs/version-3.1/v3-migration.md [hunk 2]
```diff
@@ -34,3 +34,13 @@ await TrackPlayer.updateOptions({
 -  TrackPlayer.destroy();
 -  TrackPlayer.stop();
 ```
+
+You also meed to have a minimum comile & target SDK Version for Android of 31:
```
Comment (mpivchev) [inline docs/versioned_docs/version-3.1/v3-migration.md:38]:
```suggestion
You also meed to have a minimum compile & target SDK of 31 (Android 12)
```

File: docs/docs/v3-migration.md [hunk 3]
```diff
@@ -34,3 +34,13 @@ await TrackPlayer.updateOptions({
 -  TrackPlayer.destroy();
 -  TrackPlayer.stop();
 ```
+
+You also meed to have a minimum compile & target SDK of 31 (Android 12)
```
Comment (mpivchev) [inline docs/docs/v3-migration.md:38]:
```suggestion
You also beed to have a minimum compile & target SDK of 31 (Android 12)
```

**Review by mpivchev — DISMISSED** [review]
Found one more typo but LGTM otherwise!

**Comment by jspizziri** [comment]
@mpivchev dang, I need to learn how to spell 😆 . Need another approval to merge.

**Review by mpivchev — APPROVED** [review]

**Pull request merged**